package.json

```diff
@@ -130,10 +131,53 @@
     "shx": "^0.3.2",
     "sloc": "^0.3.1",
     "sort-package-json": "^2.8.0",
-    "typescript": "~5.2.0"
+    "typescript": "~5.2.0",
+    "wireit": "^0.14.9"
   },
   "packageManager": "yarn@3.8.1",
   "engines": {
     "node": "18 || 20"
+  },
+  "wireit": {
+    "prettier:check": {
+      "command": "node ./scripts/log-output.js --log ./.tasks/prettier-check.log -- prettier --check .",
+      "files": [
+        "**/*.{.dockerbuild,.dockerfile,css,html,js,json,jsx,md,mdx,mustache,ps1,scss,sh,ts,tsx,xml,yaml,yml}"
+      ],
+      "output": [
+        "./.tasks/prettier-check.log"
+      ]
+    },
+    "prettier:fix": {
+      "command": "node ./scripts/log-output.js --log ./.tasks/prettier-fix.log -- prettier --write .",
+      "files": [
+        "**/*.{.dockerbuild,.dockerfile,css,html,js,json,jsx,md,mdx,mustache,ps1,scss,sh,ts,tsx,xml,yaml,yml}"
+      ],
+      "output": [
+        "./.tasks/prettier-fix.log"
+      ]
```

Comment:
Doesn't this need to be the actual files that where changed by prettier, or I assume those cached changes won't be applied?